resources/email-templates/account-deleted.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block title %}Account Deletion Notice{% endblock %}
+
+{% block header %}{{ username }} account has been deleted.{% endblock %}
+
+{% block content %}
+<p>
+    If you're unsure why your account was deleted,
+    please contact the administrator of Kamu Node or reach out to us at
+    <a href="mailto:admin@kamu.dev">admin@kamu.dev</a>.
```

Comment:
`support@kamu.dev` - this one exists, not sure on `admin@kamu.dev`

Reply:
`admin@kamu.dev` exist as well, but I like your option more

Changed